internal/kafka/internal/kafkas/types/types.go

```diff
@@ -7,6 +7,7 @@ type KafkaInstanceType string
 const (
 	DEVELOPER KafkaInstanceType = "developer"
 	STANDARD  KafkaInstanceType = "standard"
+	EVAL      KafkaInstanceType = "EVAL"
```

Comment:
I think the value for this should be lowercase isn't it?

Reply:
yes. fixed. thanks